skyportal/models.py

```diff
@@ -27,6 +32,18 @@
 PHOT_ZP = 23.9
 PHOT_SYS = 'ab'
 
+env, cfg = load_env()
```

Comment:
These changes exist in the spatial backend PR -- should be separated out

Reply:
This PR is based on that one